docs-ref-conceptual/azure-cli-sp-tutorial-5.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Work with Azure service principals using a certificate – Azure CLI | Microsoft Docs
+description:  Use service principals with an existing certificate to gain control over which Azure resources can be accessed.
+manager: jasongroce
+author: daphnemamsft
+ms.author: daphnema
+ms.date: 08/16/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure service principal, create service principal azure, create service principal azure cli
+---
+
+# 5 - Using Service Principal with certificate-based authentication
```

Comment:
Let's change this to something like `Create a service principal using certificate-based authentication`